### PR TITLE
Remove cfy local command

### PR DESCRIPTION
**Pull request by aryanbaghi**
local command removed

### CODE REVIEW

**Comment by bmansheim** [comment]
Hi Aryan. Thanks for your PR!

I notice that you based it on Cloudify 3.4.1. Is that intentional or is the fix to the docs intended for a more recent version?

Thanks!

Ben

**Comment by aryanbaghi** [comment]
Hi Ben,
This is my Cloudify version
Cloudify CLI 17.11.22

**Comment by bmansheim** [comment]
Thanks, Aryan. I'll take care of the merging this fix to the correct branch.

Would it be helpful for you to use the docs at: docs.cloudify.co/community? This is the location where we update docs for the community versions.

File: content/intro/getting-started.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ Once downloaded, extract the file to a directory and cd into it.
 Now let's prepare the environment for our application:
 
 {{< gsHighlight  bash >}}
-$ cfy local init --blueprint-path blueprint.yaml --inputs '{"webserver_port": "8000", "host_ip":"localhost"}'
+$ cfy install init blueprint.yaml --inputs '{"webserver_port": "8000", "host_ip":"localhost"}'
```
Comment (isaac-s) [inline content/intro/getting-started.md]: I believe this should be `cfy init`, not `cfy install init`

**Comment by bmansheim** [comment]
Implemented in master in: https://github.com/cloudify-cosmo/docs.getcloudify.org/commit/d86846264161b151a0ed2082f5ae76377a759d27

**Pull request closed without merging**